### PR TITLE
VIM 3848: Removes some redundancy from VideoSettings

### PR DESCRIPTION
**Pull request by cjlarsen**
<b>Ticket</b>
https://vimean.atlassian.net/browse/VIM-3848

<b>Ticket Summary</b>
"Something strange occurred" alert when you upload video with just spaces in title

<b>Implementation Summary</b>

Trim whitespace from title before starting upload
How To Test
1. open the app 
2. upload your video 
3. tap on a video and set your title with only spaces 
4. tap [upload]
5. Title is now "Untitled"

### CODE REVIEW

**Comment by cjlarsen** [comment]
@alfiehanssen Ready to review and discuss, thanks!

File: VimeoUpload/Upload/Model/VideoSettings.swift [hunk 1]
```diff
@@ -55,8 +55,12 @@ class VideoSettings: NSObject
         self.privacy = privacy
         self.users = users
         self.password = password
-        self.title = self.trim(title)
-        self.desc = self.trim(description)
+        
+        // Wrap in closure so didSet gets called [CL] 3/25/16
+        ({
```
Comment (alfiehanssen) [inline VimeoUpload/Upload/Model/VideoSettings.swift]: Hey @cjlarsen I saw your comment on the other PR and didn't remember that the setter wouldn't be called in init. I prefer your original approach to the unnamed closure. I think you should revert to just calling `self.trim` in init.

**Comment by alfiehanssen** [comment]
:+1: Good to merge

File: VimeoUpload/Upload/Model/VideoSettings.swift (hunk 1 above)
Comment (cjlarsen) [inline VimeoUpload/Upload/Model/VideoSettings.swift]: @alfiehanssen Cool, will revert and merge.  Thanks!

**Pull request merged**